Realm/Realm.UnityWeaver/UnityWeaver.cs

```diff
@@ -284,26 +284,34 @@ public void OnPostBuildPlayerScriptDLLs(BuildReport report)
 
         public void OnPostprocessBuild(BuildReport report)
         {
-            if (report == null || report.summary.platform != BuildTarget.iOS)
+            switch (report?.summary.platform)
             {
-                return;
+                case BuildTarget.iOS:
+                case BuildTarget.tvOS:
+                    UpdateiOSFrameworks(false, false, report.summary.platform);
+                    break;
             }
-
-            UpdateiOSFrameworks(
-                enableForDevice: false,
-                enableForSimulator: false);
         }
 
         public void OnPreprocessBuild(BuildReport report)
```

Comment:
Similarly to iOS, we need to include the correct framework depending on the target we're building.